projectile.el

```diff
@@ -950,7 +951,11 @@ Files are returned as relative paths to the project root."
   :group 'projectile
   :type 'string)
 
-(defcustom projectile-fossil-command "fossil ls | tr '\\n' '\\0'"
+(defcustom projectile-fossil-command (concat "fossil ls | "
+                                             (when (string-equal system-type
```

Comment:
If you think that would be better I can change it. I decided on `concat` to avoid duplicating the lion's share of the string.